projects/demo/src/modules/components/rating/rating.template.html

```diff
@@ -0,0 +1,149 @@
+<tui-doc-page header="Rating" package="KIT" type="components">
+    <ng-template pageTab>
+        <p i18n>A rating indicates user interest in content</p>
+
+        <tui-doc-example
+            id="base"
+            i18n-heading
+            heading="Basic"
+            [content]="example1"
+        >
+            <tui-rating-example-1></tui-rating-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="colors"
+            i18n-heading
+            heading="Customization"
+            [content]="example2"
+        >
+            <tui-rating-example-2></tui-rating-example-2>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <b>Current rating</b>
+            - {{rating.value}}
+            <br />
+
+            <tui-rating
+                #rating
+                [min]="min"
+                [max]="max"
+                [style.color]="color"
+                [disabled]="disabled"
+                [readOnly]="readOnly"
+                [iconNormal]="iconNormal"
+                [iconFilled]="iconFilled"
+                [(ngModel)]="value"
```

Comment:
In our examples we use `FormControl` instead of `[(ngModel)]` and pass it to `tui-doc-demo`, see some other page about a control. It is necessary for this part to work:
![image](https://user-images.githubusercontent.com/11832552/148948025-5a91d39e-aa80-42a0-892b-2ec8322d9f38.png)

Reply:
fixed